src/ModelContextProtocol/Client/SseClientTransportOptions.cs

```diff
@@ -5,12 +5,14 @@ namespace ModelContextProtocol.Client;
 /// </summary>
 public record SseClientTransportOptions
 {
+    private Uri field = default!;
+    
```

Comment:
```suggestion
```

Undo the related changes in the Endpoint getter and undo the `field` changes in StdioClientTransportOptions too.